include/ntop_defines.h

```diff
@@ -524,6 +524,7 @@
 #define ALERTS_MANAGER_NOTIFICATION_ENABLED    "ntopng.alerts.notification_enabled"
 #define ALERTS_MANAGER_NOTIFICATION_SENDER     "ntopng.alerts.sender_username"
 #define ALERTS_MANAGER_NOTIFICATION_WEBHOOK    "ntopng.alerts.slack_webhook"
+#define ALERTS_MANAGER_NOTIFICATION_GRAVITY    "ntopng.prefs.slack_alert_gravity"
```

Comment:
gravity is awkward has another meaning.

use word 'level' or 'severity' that are the standard terms ntopng uses.